models/Changelog.js

```diff
@@ -0,0 +1,15 @@
+import mongoose from "mongoose";
+ 
+const ChangelogSchema = new mongoose.Schema(
+  {
+    username: String,
```

Comment:
Sorry to make another change, could the `user._id` be added to the field `userId` in this model please

Reply:
Oh, that's true.

I'll appropriate.

Reply:
thank you 👍

Reply:
Hi @eddiejaoude, I hope you've been well.

I couldn't figure out how to get the `userId`, I have attributed this to my limited knowledge of what the `id` looks like. But it is important to state that I found a certain `id` in the `session` object...

```shell
user: {
  "name": "Olabode Lawal-Shittabey",
  "email": "babblebey@gmail.com",
  "image": "https://avatars.githubusercontent.com/u/25631971?v=4",
  "id": 2563**** // <- THAT ID!?
}
expires: "2023-10-11T17:56:10.427Z"
accessToken: "g******u49Hv5n***************DR******i"
username: "babblebey"
```

...but its a `8 character` id, I obviously think that the `userId` was going to match the character length of the `mongoose.Schema.Types.ObjectId` type as seen in the `users` and `profiles` collection in the `db`. 

Kindly confirm, if this is the `id` in `session.user.id` that you had in mind that we use or is there a part I should be looking at for the id?